src/common/src/array/arrow/arrow_impl.rs

```diff
@@ -413,10 +413,12 @@ pub trait ToArrow {
 
     #[inline]
     fn decimal_type_to_arrow(&self, name: &str) -> arrow_schema::Field {
-        arrow_schema::Field::new(name, arrow_schema::DataType::Utf8, true)
-            .with_metadata([("ARROW:extension:name".into(), "arrowudf.decimal".into())].into())
```

Comment:
I remember UDF will encode decimal to utf8. This is a breaking change 🥵

Reply:
I thought it was a bug, so let's do not change the arrowudf part, but only changed the first place.